### PR TITLE
Prepared Key Vault for the April 2021 beta release

### PR DESCRIPTION
**Pull request by vcolin7**

### CODE REVIEW

File: sdk/keyvault/azure-security-keyvault-administration/README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Maven dependency for the Azure Key Vault Administration library. Add it to your
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-security-keyvault-administration</artifactId>
-    <version>4.0.0-beta.5</version>
+    <version>4.0.0-beta.6</version>
```
Comment (alzimmermsft) [inline sdk/keyvault/azure-security-keyvault-administration/README.md:17]: Is this version released?

File: sdk/keyvault/azure-security-keyvault-certificates/README.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ Maven dependency for the Azure Key Vault Certificate client library. Add it to y
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-security-keyvault-certificates</artifactId>
-    <version>4.2.0-beta.4</version>
+    <version>4.2.0-beta.5</version>
```
Comment (alzimmermsft) [inline sdk/keyvault/azure-security-keyvault-certificates/README.md:17]: Is this version released?

File: sdk/keyvault/azure-security-keyvault-secrets/README.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ Maven dependency for the Azure Key Vault Secrets client library. Add it to your
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-security-keyvault-secrets</artifactId>
-    <version>4.3.0-beta.4</version>
+    <version>4.3.0-beta.5</version>
```
Comment (alzimmermsft) [inline sdk/keyvault/azure-security-keyvault-secrets/README.md:19]: Is this version released?

File: sdk/keyvault/azure-security-keyvault-keys/README.md [hunk 4]
```diff
@@ -14,7 +14,7 @@ Maven dependency for the Azure Key Vault Key client library. Add it to your proj
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-security-keyvault-keys</artifactId>
-    <version>4.3.0-beta.5</version>
+    <version>4.3.0-beta.6</version>
```
Comment (alzimmermsft) [inline sdk/keyvault/azure-security-keyvault-keys/README.md:17]: Is this version released?

**Review by alzimmermsft — APPROVED** [review]

File: sdk/keyvault/azure-security-keyvault-administration/README.md (hunk 1 above)
Comment (vcolin7) [reply sdk/keyvault/azure-security-keyvault-administration/README.md:17]: We are releasing these versions. I updated the READMEs so they would be mentioned in the generated docs.

File: sdk/keyvault/azure-security-keyvault-certificates/README.md (hunk 2 above)
Comment (vcolin7) [reply sdk/keyvault/azure-security-keyvault-certificates/README.md:17]: Please refer to [this comment](https://github.com/Azure/azure-sdk-for-java/pull/20521#discussion_r610940121).

File: sdk/keyvault/azure-security-keyvault-secrets/README.md (hunk 3 above)
Comment (vcolin7) [reply sdk/keyvault/azure-security-keyvault-secrets/README.md:19]: Please refer to [this comment](https://github.com/Azure/azure-sdk-for-java/pull/20521#discussion_r610940121).

File: sdk/keyvault/azure-security-keyvault-keys/README.md (hunk 4 above)
Comment (vcolin7) [reply sdk/keyvault/azure-security-keyvault-keys/README.md:17]: Please refer to [this comment](https://github.com/Azure/azure-sdk-for-java/pull/20521#discussion_r610940121).

File: sdk/keyvault/azure-security-keyvault-administration/README.md (hunk 1 above)
Comment (vcolin7) [reply sdk/keyvault/azure-security-keyvault-administration/README.md:17]: I will update to the GA versions after the beta release. Last month I forgot to do that but that's how it should look like.

**Pull request merged**